packages/build-utils/src/fs/run-user-scripts.ts

```diff
@@ -123,24 +129,24 @@ export function execAsync(
 
       child.on('error', reject);
       child.on('close', (code, signal) => {
-        if (code !== 0) {
-          const cmd = opts.prettyCommand
-            ? `Command "${opts.prettyCommand}"`
-            : 'Command';
-
-          return reject(
-            new NowBuildError({
-              code: `BUILD_UTILS_EXEC_${code || signal}`,
-              message: `${cmd} exited with ${code || signal}`,
-            })
-          );
+        if (code === 0 || opts.returnInsteadOfThrowing) {
```

Comment:
Hmm, I might rename `returnInsteadOfThrowing`. I just want to handle the case where the process exits with some other code than 0, not other types of errors.

Reply:
Is there any precendent for that in `execa` or others? Trying to think of a better name here.

I know there is a [`reject: false`](https://www.npmjs.com/package/execa#reject) that works the way I'm describing: 

> resolves the promise with the error instead of rejecting it.